src/views/Simulation/components/InspectorDrawer/components/InspectorChart.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import * as d3 from 'd3';
+
+const MARGIN = { top: 10, right: 5, bottom: 40, left: 5 };
+
+export const InspectorChart = ({ chartColor = '#40E0D0', data, width, height }) => {
+  const svgRef = useRef();
+
+  useEffect(() => {
+    if (data == null) return;
+    const chartWidth = width - MARGIN.left - MARGIN.right;
```

Comment:
Because props _width_ and height are not _required_, does any error occur if one of them are null or undefined ?